Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -278,27 +261,16 @@ import Foundation
     ///   HTTP response and `error` will be `nil`; on failure, `body` and `response` will be
     ///   `nil` and `error` will contain the error that occurred.
     @_documentation(visibility: private)
-    @objc(POST:parameters:httpType:completion:)
+    @objc(POST:parameters:httpType:configuration:completion:)
     public func post(
         _ path: String,
         parameters: [String: Any]? = nil,
         httpType: BTAPIClientHTTPService = .gateway,
+        configuration: BTConfiguration? = nil,
```

Comment:
this is very clean! We are assuming same config per user action like tokenize, right?